pkg/snapshot/overlay.go

```diff
@@ -1160,6 +1169,21 @@ func (o *snapshotter) createSnapshot(ctx context.Context, kind snapshots.Kind, k
 		}
 	}
 
+	_, info, _, err := storage.GetInfo(ctx, key)
```

Comment:
check this err?

Reply:
I think 'image_ref' can be saved after info get in Line 1193?
![image](https://github.com/containerd/accelerated-container-image/assets/5982966/916e9484-5d71-4e87-8f07-9c495fb2c2cd)